libcxx/docs/ReleaseNotes/19.rst

```diff
@@ -130,9 +151,9 @@ Deprecations and Removals
   as ``std::vector<const T>`` and ``std::list<const T>``. This used to be supported as an undocumented extension.
   If you were using ``std::vector<const T>``, replace it with ``std::vector<T>`` instead. The
   ``_LIBCPP_ENABLE_REMOVED_ALLOCATOR_CONST`` macro can be defined to temporarily re-enable this extension as
-  folks transition their code. This macro will be honored for one released and ignored starting in LLVM 20.
+  to transition to update user code. This macro will be honored for one released and ignored starting in LLVM 20.
```

Comment:
`to temporarily re-enable this extension to make it easier to update user code`?